app/assets/javascripts/application.js

```diff
@@ -13,4 +13,6 @@
 //= require rails-ujs
 //= require activestorage
 //= require turbolinks
-//= require_tree .
+//= require chartkick
+//= require Chart.bundle
+//= require_tree .
```

Comment:
The newline character is missing again, I suggest changing your editor's settings or something to detect them 🤔